CONTRIBUTING.md

```diff
@@ -13,3 +13,30 @@ the [CC0 1.0 Universal public domain dedication](https://creativecommons.org/pub
 All contributions to this project will be released under the CC0
 dedication. By submitting a pull request, you are agreeing to comply
 with this waiver of copyright interest.
+
+## Starting Work on an Issue
+
+Issues that are marked with the `ready` label are ripe for the picking!  Simply assign yourself to the issue to and change its label to `in progress` to indicate that you are working on it.
+
+If the issue involves writing code or producing some other change that will result in a pull request, begin by creating yourself a branch with a short descriptive name of the work that includes the issue number at the end, e.g., `document-pr-process-#36` - if you are not a part of the 18F Team, please fork the repository first and then create a branch for yourself with the same convention.
+
+Once your local branch is created, simply push it remotely and this will assign the issue to you and move it to be `in progress` automatically.
+
+## Submitting a Pull Request and Completing Work
+
+When you are satisfied with your work and ready to submit it to be completed, please submit a pull request for review.  If you haven't already, please follow the instructions above and create a branch for yourself first.  Prior to submitting the pull request, please make note of the following:
+
+1. Code changes should be accompanied by tests.
+2. Please run the tests (`$ ./go test`) to make sure there are no regressions.
+
+Once everything is ready to go, [submit your pull request](https://help.github.com/articles/using-pull-requests/)!  When creating a pull request please be sure to reference the issue number it is associated with, preferably in the title.
+
+If you are a part of the 18F team you can reference the issue like this:  `Closes #36: Short sentence describing the pull request`
+
+If you are not a part of the 18F team, please reference the issue like this:  `Closes 18F/about_yml#36:  Short sentence describing the pull request`
```

Comment:
I think it might be clearer here to say "If you are working in a forked repo" (and possibly "If you are working in a branch of the main about_yml repo" above).